projects/core/src/error-handling/utils/error-interceptor-utils.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ChainedErrorInterceptorFn,
+  ErrorInterceptor,
+  ErrorInterceptorPriority,
+} from '../error-interceptors/error-interceptor';
+
+/**
+ * Sorts error interceptors based on priority.
```

Comment:
[MINOR]
Let's change this doc, only when we agree on the strategy of storting
```suggestion
 * Sorts error interceptors based on priority. Preserves the original order within a group of interceptors with the same priority.
```